### PR TITLE
[cgnscheck] check multiple gridcoordinates nodes & nullpointer

### PR DESCRIPTION
**Pull request by ndelling**
Hi,

this branch includes improvements of cgnscheck:
- check all GridCoordinates_t nodes
- allow "Null" keywords for *Pointers

### CODE REVIEW

**Comment by MicK7** [comment]
LGTM

Waiting for @brtnfld approval

**Review by brtnfld — APPROVED** [review]
LGTM

**Pull request merged**
